be/src/io/s3_input_stream.cpp

```diff
@@ -30,7 +30,9 @@ StatusOr<int64_t> S3InputStream::read(void* out, int64_t count) {
         _offset += body.gcount();
         return body.gcount();
     } else {
-        return Status::IOError(outcome.GetError().GetMessage());
+        const auto& error = outcome.GetError();
+        return Status::IOError(fmt::format("code={}(SdkErrorType:{}), message={}", error.GetResponseCode(),
```

Comment:
extract as a function. us can be used in multiple place

Reply:
OK. Fixed.